include/common.hpp

```diff
@@ -4,13 +4,15 @@
 
 #include <cstdlib>
 #include <vector>
+#include <Eigen/Dense>
 
 
 namespace GQCG {
 
 
 typedef std::vector<size_t> Vectoru;
 typedef std::vector<Vectoru> Matrixu;
+typedef Eigen::Matrix<size_t,Eigen::Dynamic,1> VectorXs;
```

Comment:
do we still need `Vectoru` if we have this?

use modern-style type aliases: `using VectorXs = Eigen::Matrix<size_t, Eigen::Dynamic, 1>`